spec/eof.md

```diff
@@ -299,6 +299,7 @@ The following instructions are introduced in EOF code:
 - no unreachable code sections are allowed, i.e. every code section can be reached from the 0th code section with a series of CALLF / JUMPF instructions, and section 0 is implicitly reachable.
 - it is an error for a container to contain both `RETURNCONTRACT` and either of `RETURN` or `STOP`.
 - it is an error for a subcontainer to never be referenced in code sections of its parent container
+- it is an error for a given subcontainer to be referenced by both `RETURNCONTRACT` and `EOFCREATE`
 - for terminology purposes, the following concepts are defined:
     - an "initcode" container is one which does not contain `RETURN` or `STOP`
     - a "runtime" container is one which does not contain `RETURNCONTRACT`
```

Comment:
What I really would like is to get rid of the line below, that defines "initcode_runtime" containers... It notes the corner case, but doesn't add any new rule, and I think it's just confusing.

So now I'm starting to feel that it's better to get rid of "initcode" and "runtime" definitions after all. Or redefine them as "container referenced by EOFCREATE/creation tx" and "container referenced by RETURNCONTRACT" ? I'm not sure.

Reply:
I think the note was useful to ourselves, to remind us of the corner case. I agree that at this point we can just better drop it.

I would still favor removing the definitions, not reintroducing different ones. I have more problem with their names and existence, not in what they contain.

Reply:
Okay let's remove them.

Reply:
OK, but I think I'll open a third PR for removing definitions, mixing an editorial change with spec change in #129 was a mistake. Hang on...